ams/server/src/main/java/com/netease/arctic/server/table/TableRuntime.java

```diff
@@ -171,6 +171,7 @@ public void setPendingInput(OptimizingEvaluator.PendingInput pendingInput) {
       this.pendingInput = pendingInput;
       if (optimizingStatus == OptimizingStatus.IDLE) {
         this.currentChangeSnapshotId = System.currentTimeMillis();
+        this.currentStatusStartTime = System.currentTimeMillis();
```

Comment:
If `currentStatusStartTime` and `optimizingStatus` always change together, would it be better to abstract a method to encapsulate their changes?

Reply:
Yes, it is. I will extract a method to handle the change.